components/bingx/actions/custom-bracket-order/custom-bracket-order.mjs

```diff
@@ -2,7 +2,7 @@ import bingx from "../../bingx.app.mjs";
 
 export default {
   name: "BingX Custom Bracket Order",
-  version: "0.0.3",
+  version: "0.0.4",
```

Comment:
A new feature/addition should warrant a new `minor` version [as per the guidelines](https://pipedream.com/docs/components/guidelines/#versioning), the same goes for the other component as well as the package version.
```suggestion
  version: "0.1.0",
```